examples/render/.terraform.lock.hcl

```diff
@@ -0,0 +1,24 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/render-oss/render" {
```

Comment:
Should this file be gitignored? I don't think lock files are normally in examples

Reply:
Removed. I'll update .gitignore in a follow up PR